### DISCUSSION TITLE
Autocomplete to suggest variable

### DISCUSSION THREAD

**Discussion by cyriltovena** [grafana/grafana#65890] [Dashboards, Panels & Visualizations]
I always write a dashboard with variable, but when writing the PromQL query  I want to have in the autocomplete for label selector values as option to select one of the variable for the current dashboard.

<img width="1692" alt="image" src="https://user-images.githubusercontent.com/1053421/229725470-45864963-46d9-41c9-9126-85e5f5dbab90.png">

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82767. Feel free to continue the discussion around this there. Thank you!
